test/uk/gov/hmrc/merchandiseinbaggage/service/CalculationServiceSpec.scala

```diff
@@ -146,20 +146,20 @@ class CalculationServiceSpec extends BaseSpecWithApplication with ScalaFutures w
     val importGoods                     = aImportGoods
     val results: Seq[CalculationResult] =
       Seq(
-        CalculationResult(importGoods, AmountInPence(100000), AmountInPence(0), AmountInPence(0), None),
```

Comment:
Any reason, why this was updated? It is not necessary as it is already within the threshold plus each gbpAmount value is overridden to an amount over the threshold. Also, on line 145, can you please update the test name to specify Import Goods, thanks.